samples/secrets-manager/infrastructure/.gitignore

```diff
@@ -0,0 +1,11 @@
+*.js
```

Comment:
can we move this file to the .gitignore file in the root folder? Otherwise, we will end up haiving many .gitignore files